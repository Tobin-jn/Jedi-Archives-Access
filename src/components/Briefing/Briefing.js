import React from 'react'

import './Briefing.css'

const Briefing = ( {crawl, title, id}) => {
  return (
    <section>
      <h2>Situation Briefing</h2>
      <p className='crawl-text'>{crawl}</p>
      <p className='smol-text'>Briefing Title: <span>{title}</span></p>
      <p className='smol-text'>Galaxy Date: <span>{id}</span></p>
    </section>
  )
}

export default Briefing;