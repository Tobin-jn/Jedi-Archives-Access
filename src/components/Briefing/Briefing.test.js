import React from 'react';
import Briefing from './Briefing';

import { shallow } from 'enzyme';

let wrapper;


beforeEach(() => {
  wrapper = shallow(<Briefing crawl='' title='' id=''

  />)
});

describe('Briefing', () => {
  it ('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});