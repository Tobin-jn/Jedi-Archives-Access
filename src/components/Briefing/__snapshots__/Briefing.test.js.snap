// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Briefing should match the snapshot 1`] = `
<section>
  <h2>
    Situation Briefing
  </h2>
  <p />
  <p
    className="smol-text"
  >
    Briefing Title: 
    <span />
  </p>
  <p
    className="smol-text"
  >
    Galaxy Date: 
    <span />
  </p>
</section>
`;
